docs/docsServer.js

```diff
@@ -1,25 +1,40 @@
 const esbuild = require("esbuild");
 const chokidar = require("chokidar");
 const liveServer = require("live-server");
-const { config: devScriptBuildConfig } = require("./buildConfigs");
+const { config: docsBuildConfig } = require("./buildConfigs");
+const { esm: packageBuildConfig } = require("../buildConfigs");
+
+// Update chat package directories since this script is executing in a subdirectory.
```

Comment:
What does "Update chat package directories" mean? Does it mean "Point to different output directories"?

Reply:
https://github.com/vectara/react-chatbot/pull/9/commits/d439e717267622629e3ea5c4983c52e9dfa59d51